python/tests/integration/completions/test_azure_oai_chat_service_extensions.py

```diff
@@ -126,7 +136,7 @@ async def create_with_data_chat_function(get_aoai_config, create_kernel, create_
 
 
 @pytest.mark.asyncio
-@pytest.mark.skip(reason="Azure OpenAI Chat Completion with extensions is not working")
+@pytestmark
```

Comment:
no, this is set to skip if either the package are not installed (checked by trying to import azurecogsearchmemory store) or because the env values are not there, if those are then then this lets the test execute, if those are missing it is marked for skip.